addOns/oast/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Fixed
+- OAST Interactsh Options Dialog: If host or token config changed the 'New Payload' Button generates the Payload still with the old config.  
```

Comment:
It's not clear what the fix is though, maybe add that the button is disabled in that case. It might worth noting this in the help page.